.dev/bin/comment-on-pr.sh

```diff
@@ -50,7 +50,7 @@ BOT_COMMENTS=$(curl -sS \
   -H "Accept: application/vnd.github.v3+json" \
   "https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/issues/$PR_ID/comments")
 
-COMMENT=$(echo $BOT_COMMENTS | jq '[.[] | select( .user.login == "godaddy-wordpress-bot" and ( .body | contains( "Code Coverage" ) | not ) )]' | jq '.[-1]')
+COMMENT=$(echo $BOT_COMMENTS | jq '[.[] | select( .user.login == "godaddy-wordpress-bot" and ( .body | contains( "Download" ) ) )]' | jq '.[-1]')
```

Comment:
This is unfamiliar to me. I suppose we are looking perviously for 'code coverage' but that is now changed to 'dowload'. Do you mind explaning why this change was made?

Reply:
@AnthonyLedesma That should not be "Download" I'm not sure why that was changed. Good catch.

Reply:
@AnthonyLedesma My mistake. I thought this was the other `comment-perf-results-on-pr.sh`. This file is the script that posts the CircleCI artifact link (the CoBlocks build .zip) as a comment on GitHub. There are three comments being posted on each PR. The Download link to the .zip from CircleCI, the Coverage Results and now the Performance Test Results.

Previously, I was just checking for any comment by the `godaddy-wordpress-bot` that didn't contain "Code Coverage", but now that is another comment being posted I specifically am searching for the text "Download" in the comment.

Reply:
Just for my clarity, this needs to be reverted is that correct?